istioctl/pkg/injector/injector-list.go

```diff
@@ -125,7 +125,7 @@ func injectorListCommand(ctx cli.Context) *cobra.Command {
 func filterSystemNamespaces(nss []corev1.Namespace, istioNamespace string) []corev1.Namespace {
 	filtered := make([]corev1.Namespace, 0)
 	for _, ns := range nss {
-		if inject.IgnoredNamespaces.Contains(ns.Name) || ns.Name == istioNamespace {
+		if common.IgnoredNamespaces.Contains(ns.Name) || ns.Name == istioNamespace {
```

Comment:
maybe we should use annotation/label instead of environment variable?

Reply:
Could you elaborate more on this idea? I am not familiar with `istioctl experimental injector list`.

Reply:
sorry, I put my comment on wrong line

Reply:
Ok, so if you mean replacing the env var entirely with an annotation, I don't like this idea... It would be much less convenient, if user had to annotate namespace first and then deploy ambient mesh.

Reply:
+1, our installer cannot really add it automatically either

Reply:
not sure if it's a good idea to add annotation when namespace controll started?
if I cannot find something should be there, the annotaion wil tell me that's expected.

Reply:
Oh I misunderstood. I thought you meant use annotation to opt a namespace in/out. But you mean for istioctl.

Istioctl depending on internal feature flag variables like INJECTION_IGNORED_NAMESPACES is basically broken; these are meant to be set in istiod, not on user's laptops when they run istioctl. This effectively means istioctl will always use the defaults I guess...

Reply:
+1 - experimental feature flags in istiod, certainly not for istioctl which has its own experimental flags and CLI.

And if it's install related - it wouldn't help in helm. 

Helm values.yaml maybe another quasi-API that is not never properly reviewed.

Reply:
> Istioctl depending on internal feature flag variables like INJECTION_IGNORED_NAMESPACES is basically broken;

Yes, this is why I didn't use `features.InjectionIgnoredNamespaces` and I kept the default ignored namespaces, but this would not work correctly when ignored namespaces are customized...